timed/projects/tests/test_cost_center.py

```diff
@@ -1,15 +1,35 @@
+import pytest
 from django.urls import reverse
-from rest_framework.status import HTTP_200_OK
+from rest_framework.status import HTTP_200_OK, HTTP_403_FORBIDDEN
 
-from timed.projects.factories import CostCenterFactory
+from timed.employment.factories import EmploymentFactory
+from timed.projects.factories import CostCenterFactory, CustomerAssigneeFactory
 
 
-def test_cost_center_list(internal_employee_client):
+@pytest.mark.parametrize(
+    "is_employed, is_customer_assignee, is_customer, status_code",
+    [
+        (False, True, False, HTTP_403_FORBIDDEN),
+        (False, True, True, HTTP_403_FORBIDDEN),
+        (True, False, False, HTTP_200_OK),
+        (True, True, False, HTTP_200_OK),
+        (True, True, True, HTTP_200_OK),
+    ],
+)
+def test_cost_center_list(
+    auth_client, is_employed, is_customer_assignee, is_customer, status_code
+):
+    user = auth_client.user
+    if is_customer_assignee:
+        CustomerAssigneeFactory.create(user=user, is_customer=is_customer)
+    if is_employed:
+        EmploymentFactory.create(user=user)
```

Comment:
Since you're using this exact pattern all over the place: Factor it out into it's own test helper method:
```python
def setup_customer_and_employment_status(user, is_assignee, is_customer, is_employed):
    if is_assignee:
        CustomerAssigneeFactory.create(user=user, is_customer=is_customer)
    if is_employed:
        EmploymentFactory.create(user=user)
```